### PR TITLE
fix(theme-classic): should use plurals for category items description

### PR DESCRIPTION
**Pull request by baradusov**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

When enabling russian language doc cards showing source string for default description. I made a fix to use appropriate plural form of the description.

## Test Plan

Before:
Screenshot 2024-02-15 at 15 18 11

After:
Screenshot 2024-02-15 at 15 21 44

### Test links

Deploy preview: https://deploy-preview-9851--docusaurus-2.netlify.app/

### CODE REVIEW

File: packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx [hunk 1]
```diff
@@ -24,6 +25,23 @@ import type {
...
 
+function useCategoryItemsPlural() {
+  const {selectMessage} = usePluralForm();
+  return (count: number) =>
+    selectMessage(
+      count,
+      translate(
+        {
+          message: '{count} items',
+          id: 'theme.docs.DocCard.categoryDescription',
```
Comment (slorber) [inline packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx:36]: Could you add a `.plurals` suffix to this id?

We usually have this convention. 

Also need to rename all existing keys in `docusaurus-theme-translations`.

**Review by slorber — CHANGES_REQUESTED** [review]
Thanks

Yes it should support plural forms.

Unfortunately other languages will have to be fixed too because the string does not contain all plural forms, but that's not your responsibility to fix, and other languages should keep working as is (ie render as before a possibly wrong label)

---

BTW if you speak Russian, it would be a good opportunity to check the existing translations, notably this one missing plural forms:

```
  "theme.blog.post.readingTime.plurals": "{readingTime} мин. чтения",
```

**Comment by baradusov** [comment]
> BTW if you speak Russian, it would be a good opportunity to check the existing translations, notably this one missing plural forms:
> 
> ```
>   "theme.blog.post.readingTime.plurals": "{readingTime} мин. чтения",
> ```

@slorber I checked all russian translations and everything seems okay to me

`"theme.blog.post.readingTime.plurals": "{readingTime} мин. чтения"`
this particular translation doesn't need plural forms, because the word that should be pluralized `мин.`(minutes) is shorthand in this case

File: packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx (hunk 1 above)
Comment (baradusov) [reply packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx]: thanks for the review
i pushed the changes you requested

**Comment by slorber** [comment]
Thanks 👍 

> `"theme.blog.post.readingTime.plurals": "{readingTime} мин. чтения"`
> this particular translation doesn't need plural forms, because the word that should be pluralized `мин.`(minutes) is shorthand in this case

Actually it's better to use 3 times the plural forms even though they are the same string. If you don't there will be a warning saying plural forms are missing, helping us actually catch that we may have forgotten them.

**Pull request merged**